lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.mel

```diff
@@ -116,7 +116,7 @@ proc fileOptionsTabPage(string $tabLayout)
     setParent $topForm;
     frameLayout -label `getMayaUsdLibString("kMayaRefDescription")`;
     columnLayout -adjustableColumn true;
-    text -align "left" -label `getMayaUsdLibString("kMayaRefAddToUSDDescription")`;
+    text -align "left" -wordWrap 1 -label `getMayaUsdLibString("kMayaRefAddToUSDDescription")`;
```

Comment:
Doesn't wordWrap flag take a boolean? Using a 1 makes it seems like its some kind of word wrap value choice.

Reply:
https://help.autodesk.com/view/MAYAUL/2022/ENU/?guid=GUID-CCECE76B-A48A-45CF-A4C1-F121004E3E7A

Reply:
No wonder it is confusing :) I guess it is all for historical reasons. Putting true here does cost much, so I did.